### PR TITLE
22 refactor validationexceptionhandler

### PR DESCRIPTION
**Pull request by misterrevo**
I do it in diffrent approach, but i think we do not need to makes a lot of classes if we can map key in private method,. In comment give me real names for keys (SIZE -> REAL_NAME, NOT_BLANK -> REAL_NAME, ETC.) @BarSwi

### CODE REVIEW

**Review by Luzu4 — DISMISSED** [review]

File: src/main/java/karuna/karuna_backend/exception/keys/FailedValidationErrorKey.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+public enum FailedValidationErrorKey {
+    SIZE,
+    PATTERN,
+    EMAIL,
+    MAX,
+    MIN,
+    NOT_NULL,
+    NOT_BLANK,
+    UNKNOWN
+}
```
Comment (BarSwi) [inline src/main/java/karuna/karuna_backend/exception/keys/FailedValidationErrorKey.java:13]: Implement a little bit more detailed keyName. After that obviously add swagger schema.

Comment (BarSwi) [reply src/main/java/karuna/karuna_backend/exception/keys/FailedValidationErrorKey.java:13]: INVALID_PATTERN
INVALID_EMAIL_PATTERN
VALUE_TOO_LONG
VALUE_TOO_SHORT
VALUE_IS_NULL
VALE_IS_BLANK

And just describe them that should do the job for now

Comment (misterrevo) [reply src/main/java/karuna/karuna_backend/exception/keys/FailedValidationErrorKey.java:24]: my is ok, or i should change?

Comment (misterrevo) [reply src/main/java/karuna/karuna_backend/exception/keys/FailedValidationErrorKey.java:24]: i fix before u comment

Comment (misterrevo) [reply src/main/java/karuna/karuna_backend/exception/keys/FailedValidationErrorKey.java:24]: Okay i fixed like u said

**Review by Luzu4 — APPROVED** [review]

**Review by BarSwi — APPROVED** [review]

**Pull request merged**